code_conventions.md

```diff
@@ -157,6 +181,40 @@ for (int i = 0; i < len; i++)
  * When calling the index `[]` operator, don't insert a space before the `[`.
 
 
+Static Typing (Python)
+--------
+Normally, Python is using dynamic typing. For example:
+~~~~~~~~~~~~~~~{.py}
+def fib(n):
+    a, b = 0, 1
+    while a < n:
+        yield a
+        a, b = b, a + b
+
+x = fib(5)
+y = fib(2.5)
+~~~~~~~~~~~~~~~
+which defines the a dynamically typed function to return the Fibonacci number.
+This will be fine for x, but what will the result be for y?
```

Comment:
IMHO: this is more of an example of duck typing: if it looks like a duck, it quacks like a duck (you can compare it), it's a duck". 
Integer vs. string manipulation (addition) could be a better example, if you need to supply an example at all.